templates/core/_init_tabs.html

```diff
@@ -2,11 +2,20 @@
 {# tab id must be {{ tabs }} (use {% with...%}) #}
 {% load i18n %}
 <script>
-
+const getCookieByName = (cookieName) => document.cookie.split('; ')
+    .find((cookie) => cookie.split('=')[0] === cookieName)?.split('=')[1] || null;
 
 $(document).ready(function() {
     tabprefix = "goto_";
 
+    // detect mobile devices using screen size
```

Comment:
i am not very confident with the detection of device type by screen resolution...

Reply:
I understand... If you have any suggestions I'll be happy to discuss them :relaxed: